src/components/ListCard/ListCard.tsx

```diff
@@ -89,9 +91,9 @@ export const ListCard = ({
             )}
           </>
         )}
-        {additionalContent && (
-          <Typography variant="body2">{additionalContent}</Typography>
-        )}
+        {badges?.map(badge => (
+          <Chip key={`${name}-badge-${badge}`} size="small" label={badge} />
```

Comment:
Should we just call them `chips` instead of badges?

Reply:
I thought about it and chips feels very internal to material UI whereas everyone knows what badges are (took me a while to find this chip component)

Reply:
In what way is the publicly-available component internal to Material UI? 😕 

I feel like since backstage and its plugins are written using Material UI, it makes more sense to use its design language than try to translate to an actual internal language/system (ours).

Reply:
I think these could be called chips, tags, badges, ... and we might change the way they're styled later so I don't want to commit to the chip name.

Reply:
Badge seems as arbitrary as chip to me...